scipy/linalg/decomp_svd.py

```diff
@@ -279,6 +286,98 @@ def diagsvd(s, M, N):
         raise ValueError("Length of s must be M or N.")
 
 
+def higher_order_svd(
+    a: npt.ArrayLike, *, full_tensor: bool = True, check_finite: bool = True
+) -> Tuple[List[numpy.ndarray], numpy.ndarray]:
+    """Higher-order SVD (HOSVD)
+
+    Factorizes the M-D tensor `a` into a list of M matrices ``U_k`` containing
+    the left singular vectors of the unfolded tensor with respect to the
+    ``k``\\ th axis and a core tensor ``S`` such that
```

Comment:
IMO this does not render well. I propose k-th instead? In case you don't know, you can see the output by clicking on the details of the CI job `ci/circleci: build_docs artifact` (https://32778-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.linalg.higher_order_svd.html?highlight=higher_order_svd#scipy.linalg.higher_order_svd)

Reply:
For me this is rendered as expected in your cited artifact?

![Doc](https://user-images.githubusercontent.com/4115822/123759038-d04d1d80-d8bf-11eb-9312-38a64f888b6d.png)

Reply:
👌  just wanted to make sure, fine with me then.